docs/sources/troubleshooting/support-bundles/index.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Learn how to send a support bundle to Grafana Labs support for troubleshooting
+keywords:
+  - grafana
+  - troubleshooting
+  - support
+  - bundles
+title: Send a support bundle to Grafana Labs support
+menutitle: Send a support bundle to support
+weight: 200
+---
+
+# Send a support bundle to Grafana Labs support
+
+When you encounter problems with your Grafana instance, you can send us a support bundle that contains information about your Grafana instance, including:
+
+- Grafana version
+- Installed plugins
+- Grafana configuration
+- Deployed database information and migrations
+
+## Available support bundle components
+
```

Comment:
@Jguer , I moved this section up so it's visible before the steps. We need an intro sentence after the heading but before the list. What does available support bundle components mean?

Reply:
Each component is a file inside of the support bundle.

Users can choose to include different information in the support bundle (for example: only settings and SAML information or only users list)

Reply:
got it, ok, adding a small suggestion.